package.json

```diff
@@ -33,7 +33,7 @@
   "dependencies": {
     "chalk": "1.1.3",
     "inquirer": "3.0.6",
-    "minimist": "1.2.0",
+    "minimist": "1.2.4",
```

Comment:
Hi, can we use `^1.2.0` or `~1.2.0` instead of pinning it down to a fixed version?

Reply:
Good suggestion